doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1272 @@
+
+Trusted Firmware-M (TF-M) samples
+---------------------------------
+
+* :ref:`tfm_psa_template` sample:
+
+  * Added support for the following attestation token fields:
+
+    * Profile definition
+    * PSA certificate reference (optional), configured using the ``SB_CONFIG_TFM_OTP_PSA_CERTIFICATE_REFERENCE`` sysbuild Kconfig option
+    * Verification service URL (optional), configured using the :kconfig:option:`CONFIG_TFM_ATTEST_VERIFICATION_SERVICE_URL` Kconfig option
+
+* :ref:`tfm_secure_peripheral_partition` sample:
+
+  * Updated documentation with information about how to access other TF-M partitions from the secure partition.
+
+Thread samples
+--------------
+
+* :ref:`ot_cli_sample` sample:
+
+  * Removed support for the nRF54H20 DK.
+
+Zigbee samples
+--------------
+
+* Removed all Zigbee samples.
+  They are now available as separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories.
+
+Wi-Fi samples
+-------------
+
+* Added support for nRF7002 EB II with nRF54 Series devices in the following samples:
+
+  * :ref:`wifi_station_sample`
+  * :ref:`wifi_scan_sample`
+  * :ref:`wifi_shell_sample`
+  * :ref:`wifi_radio_test_sd`
+  * :ref:`wifi_radio_test`
+  * :ref:`wifi_softap_sample`
+  * :ref:`wifi_shutdown_sample`
+  * :ref:`wifi_twt_sample`
+  * :ref:`wifi_offloaded_raw_tx_packet_sample`
+  * :ref:`wifi_raw_tx_packet_sample`
+  * :ref:`ble_wifi_provision`
+
+* :ref:`wifi_station_sample` sample:
+
+  * Added an ``overlay-zperf.conf`` overlay for :ref:`performance benchmarking and memory footprint analysis <wifi_sta_performance_testing_memory_footprint>`.
+
+* Radio test samples:
+
+  * Added:
+
+    * The :ref:`wifi_radio_test_sd` sample to demonstrate the Wi-Fi and Bluetooth LE radio test running on the application core.
+    * The ``wifi_radio_test get_voltage`` command to read battery voltage.
+
+  * Updated:
+
+    * The :ref:`wifi_radio_test` sample is now moved to :file:`samples/wifi/radio_test/multi_domain`.
+
+* :ref:`wifi_shell_sample` sample:
+
+  * Updated by modifying support for storing the nRF70 firmware patches in external flash using the :ref:`partition_manager`.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Added:
+
+    * Support for the nRF54L05 and nRF54L10 SoCs (emulated on nRF54L15 DK).
+    * FLPR core support for the :ref:`zephyr:nrf54l15dk_nrf54l15` and :ref:`zephyr:nrf54h20dk_nrf54h20` board targets.
+
+  * Removed the following compiler options that were set in the :kconfig:option:`CONFIG_COMPILER_OPT` Kconfig option:
+
+    * ``-fno-pie``
+    * ``-fno-pic``
+    * ``-ffunction-sections``
+    * ``-fdata-sections``
+
+    These options are enabled by default in Zephyr and do not need to be set with the dedicated Kconfig option.
+
+* :ref:`caf_sensor_manager_sample` sample:
+
+  * Added low power configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
```

Comment:
Please add:
```suggestion
  * Changed the name of Software Defined Peripherals (SDP) to High Performance Framework (HPF)
```
@masz-nordic , please check if this is the correct place (Other samples) and if description is ok

Reply:
Discussed offline, should be ok

Reply:
@magp-nordic, this is listed as an application here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications.html. So IMO we should be listing this under Application section: https://ncsdoc.z6.web.core.windows.net/PR-21339/nrf/releases_and_maturity/releases/release-notes-3.0.0.html#applications. Also, did you mean to remove the entry for CAF Sensor Manager sample: Added low power configuration for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target. in line 866?

Reply:
@umapraseeda ok, my mistake, this should be an application entry. 
And when it comes to removing the previous 866 line: sorry, that was not my intention. I thought I changed the suggestion to not remove the previous one, but apparently, I didn't.

Reply:
Added under Applications. Please check.